src/templates/sitemap.xml

```diff
@@ -0,0 +1,318 @@
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/contributors.html
+        </loc>
+        <lastmod>
+            2019-11-05
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/contributors.html"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/accessibility
+        </loc>
+        <lastmod>
+            2019-11-05
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/accessibility"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/css
+        </loc>
+        <lastmod>
+            
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/css"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/caching
+        </loc>
+        <lastmod>
+            2019-11-05
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/caching"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/ecommerce
+        </loc>
+        <lastmod>
+            2019-11-05
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/ecommerce"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/fonts
+        </loc>
+        <lastmod>
+            2019-11-05
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/fonts"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/compression
+        </loc>
+        <lastmod>
+            2019-11-05
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/compression"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/http2
+        </loc>
+        <lastmod>
+            2019-11-05
+        </lastmod>
+        
+        <xhtml:link 
+            rel="alternate" 
+            hreflang="en"
+            href="https://almanac.httparchive.org/en/2019/http2"/>
+           
+    </url>
+
+    <url>
+        <loc>
+            https://almanac.httparchive.org/en/2019/javascript
+        </loc>
+        <lastmod>
+            
```

Comment:
`en/2019/javascript.md` doesn't have a `last_modified`. I'll try and fix this up in the script.

Reply:
Actually, can we do this as a part of #317, please ? This PR is becoming heavy. :)

Reply:
Fine with me. This will be fixed for this chapter before go live anyway and a blank lastmod tag is not actually a bad way to handle it anyway.

And as you say #317 will possibly avoid this happening anyway.